pyproject.toml

```
@@ -1,6 +1,12 @@
 [build-system]
-requires = ["flit_core >=3.2,<4"]
-build-backend = "flit_core.buildapi"
+requires = [
+  "sphinx-theme-builder >= 0.2.0b2",
```

```suggestion
  "sphinx-theme-builder >= 0.2.0b2,<1",
```
Maybe limit the major versión?